interpreter/value_string.go

```diff
@@ -1077,6 +1077,11 @@ func (i StringValueIterator) Next(_ ValueIteratorContext, _ LocationRange) Value
 	return NewUnmeteredCharacterValue(i.graphemes.Str())
 }
 
+func (i StringValueIterator) HasNext() bool {
+	// TODO: Not implemented yet
```

Comment:
Might be good to implement this rather sooner than later, so we don't accumulate TODOs in the interpreter. In general, it might be good to have a tech debt issue for TODOs in the `interpreter` package that were added in the compiler feature branch.

Reply:
That makes sense. I left this here because we will run into it when we enable the tests. But yeah, I'll open a follow-up PR to add the iterators in particular.